capture.py

```diff
@@ -191,20 +193,61 @@ def startCapture(startTime, endTime):
             logger.error("ERROR: Unexpected error: Could not connect to MySql instance.")
             sys.exit()
 
-        stopCapture(db_name, connection, startTime, endTime)
+        stopCapture(db_name, connection, startTime, endTime, captureBucket, metricBucket, metricFileName)
```

Comment:
Why does stopCapture get called here?

Reply:
Earlier in the function I'm calling time.sleep(captureDuration) so if the user doesn't click End Capture manually, the start capture function will automatically call stopCapture().